### ISSUE TITLE
Does a new Alaveteli with ~100 pre-seeded requests grow better?

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli-experiments#19]
## Q: Can we help sites grow by encouraging the first 100 requests?

[Explain how you will measure this]

Long term, look at time-to-impact.
Short term, look at percentage increase in requests month on month, compared to a site that doesn't have pre-seeded requests

[What do we need to do to be able to take the measurements?]
- [x] Create list of 10 examples of requests to make, how to phrase them, why would you want to do it. What sorts of things make news stories - find out from partners.
- [x] Share with partners via blog post
- [ ] ~~Integrate list with info on bulk request tool.~~
- [ ] ~~Look at first 100 requests on WDTK - what are the common themes, what's interesting~~

**Comment by garethrees** [comment]
Some conversation happening here https://groups.google.com/a/mysociety.org/forum/#!topic/alaveteli/PAZXGNkYKrs

**Comment by garethrees** [comment]
The current state of this is that:

- We have a list of 10 generic requests that have generated news stories
- We are going to publish this as a blog post on mysociety.org
- We'll add it to the community update
- We'll send it to new partners (Turkey, Colombia, etc?) to encourage them to make similar requests on their own sites. At the earlier stage in an Alaveteli's life, its important that the site admins are making requests.

**Comment by garethrees** [comment]
https://transparencia.be/ have ~100 requests at or just after launch.

**Comment by garethrees** [comment]
No one has lots of traffic, but no requests.

Belgium are a good example illustrating a growth of traffic with requests, but they do have excellent press contacts, so that is likely to be playing a part.

![screen shot 2017-01-05 at 12 49 53](https://cloud.githubusercontent.com/assets/282788/21681102/8e6cb850-d345-11e6-89d7-9d7f59ab3604.png)

Nicaragua have had a bump in requests, but no corresponding bump in traffic.

![screen shot 2017-01-05 at 12 52 10](https://cloud.githubusercontent.com/assets/282788/21681154/d94e1b34-d345-11e6-92f5-96ceb510dd1b.png)

I think what's interesting here is that several new installs (Italy, Rwanda, Nicaragua, Turkey) had an initial flurry of promotion, but then flatlined. Belgium is the only one to appear to be continuing to grow after initial launch. I wouldn't put this down to geography or backing, as Italy have a similar organisation behind them.

In conclusion, I _think_ that the initial request base has helped keep the site at a higher level of activity.

Of the 90 confirmed users, 20% have set up tracks – there was something for new users to do when they got there:

```sql
SELECT COUNT(DISTINCT tracking_user_id) FROM track_things;
 count
-------
    18
```
